tests/test_points_allocators.py

```diff
@@ -0,0 +1,159 @@
+import unittest
+
+import torch
+from aepsych.config import Config
+from aepsych.models.inducing_point_allocators import (
+    AutoAllocator,
+    KMeansAllocator,
+    SobolAllocator,
+)
+from aepsych.models.utils import select_inducing_points
+from botorch.models import SingleTaskGP
+from botorch.models.utils.inducing_point_allocators import (
+    GreedyImprovementReduction,
+    GreedyVarianceReduction,
+    InducingPointAllocator,
+)
+from botorch.utils.sampling import draw_sobol_samples
+
+
+class TestInducingPointAllocators(unittest.TestCase):
```

Comment:
Need to test instantiating a full model and checking if the inducing points are correct: initially dummy but with the correct inducing_point_method set. Then after fitting once, inducing points are replaced with real inducing points.